### ISSUE TITLE
Tests not rendered with DBT_MANIFEST mode

### ISSUE THREAD

**Issue by fabiomx** [astronomer/astronomer-cosmos#631]
Hi,

I'm currently using Cosmos 1.2.1 and dbt-core 1.5.4 on Google Cloud Composer, with local execution mode.

When I set the load mode to DBT_MANIFEST, tests are not rendered. However, they are successfully rendered when I switch to the DBT_LS mode. Based on both the [documentation](https://astronomer.github.io/astronomer-cosmos/configuration/render-config.html) and the contents of the manifest file, I believe the tests should be rendered correctly in both modes.

I could switch to using the DBT_LS mode exclusively, but I face the issue mentioned [here](https://github.com/astronomer/astronomer-cosmos/issues/520#issuecomment-1749939379).

Below are the corresponding sections of my code:

```
project_config = ProjectConfig(
    dbt_project_path="/home/airflow/gcs/dags/dbt",
    manifest_path="/home/airflow/gcs/dags/dbt/target/manifest.json",
)

def get_render_config(schedule_tag: str):
    return RenderConfig(load_method=LoadMode.DBT_MANIFEST,
                        select=[f"tag:{schedule_tag}"]
                        )

@dag(dag_id="dbt_hourly", schedule_interval=timedelta(hours=1), default_args=default_args, catchup=False)
def dbt_hourly_dag():
    start = EmptyOperator(task_id="start")

    dbt_tg = DbtTaskGroup(
        project_config=project_config,
        profile_config=profile_config,
        execution_config=execution_config,
        default_args=default_args,
        operator_args=operator_args,
        render_config=get_render_config("hourly")

    )

    end = EmptyOperator(task_id="end")

    start >> dbt_tg >> end

```

On the other hand, on the tasks log, I can see this:
```
[2023-10-26, 20:19:25 UTC] {log.py:232} WARNING - [[34m2023-10-26T20:19:25.788+0000[0m] {[34msubprocess.py:[0m94} INFO[0m - [35m(astronomer-cosmos)[0m - [0m20:19:25  Unable to do partial parsing because saved manifest not found. Starting full parse.[0m
[2023-10-26, 20:19:25 UTC] {subprocess.py:94} INFO - [0m20:19:25  Unable to do partial parsing because saved manifest not found. Starting full parse.
```
and I can't understand it since the load mode is set to `DBT_MANIFEST`, and it could render the models thanks to the manifest file, right? Or I'm missing something? :) 

Thanks!

**Comment by fabiomx** [comment]
Ok, I've just seen this https://github.com/astronomer/astronomer-cosmos/pull/615
So I suppose we can close this issue :)

**Comment by tatiana** [comment]
Thanks for reporting the issue, @fabiomx ! This is a duplicate of #580.

We'll be releasing the fix as part of Cosmos 1.2.2, next week!
